lib/Field/value/domain.php

```
@@ -0,0 +1,183 @@
+    }
+
+    public static function getSearchField($params)
+    {
+        if (isset($params['searchform']) && $params['searchform']) {
+            $params['searchform']->setValueField('text', ['name' => $params['field']->getName(), 'label' => $params['field']->getLabel()]);
+        }
+    }
+
+    public static function getListValue($params)
+    {
+                $field = $params['field'];
```

PHPMD (2.15.0)

Avoid unused local variables such as '$field'. (undefined)

(UnusedLocalVariable)